.github/workflows/update-registry.yml

```diff
@@ -74,12 +87,12 @@ jobs:
             
             - Added chain ID ${{ inputs.chain_id }} to all contracts in version ${{ inputs.version }}
           title: |
-            Add chain ${{ inputs.chain_id }} to ${{ inputs.version }} registry
+            Add chain ${{ inputs.chain_id }} to ${{ inputs.version == 'v1.3.0-both' && 'v1.3.0 (canonical + eip155)' || inputs.version }} registry
           body: |
             This PR adds chain ID `${{ inputs.chain_id }}` to the registry for version `${{ inputs.version }}`.
             
             **Changes:**
-            - Added chain ID `${{ inputs.chain_id }}` to all contract JSON files in `src/assets/${{ inputs.version }}/`
+            ${{ inputs.version == 'v1.3.0-both' && format('- Added chain ID `{0}` to all contract JSON files in `src/assets/v1.3.0/` for both canonical and eip155 deployment types', inputs.chain_id) || format('- Added chain ID `{0}` to all contract JSON files in `src/assets/{1}/`', inputs.chain_id, inputs.version) }}
```

Comment:
This doesn't produce correct values for `v1.3.0-eip155` or `v1.3.0-canonical`, why the extra effort to make it work with `v1.3.0`?